OrbitService/Utils.cpp

```diff
@@ -139,7 +142,8 @@ ErrorMessageOr<std::vector<ModuleInfo>> ParseMaps(std::string_view proc_maps_dat
 ErrorMessageOr<std::vector<orbit_grpc_protos::TracepointInfo>> ReadTracepoints() {
   std::vector<TracepointInfo> result;
 
-  for (const auto& category : fs::directory_iterator(kLinuxTracingEvents)) {
+  for (const auto& category : fs::directory_iterator(
+           kLinuxTracingEvents, std::filesystem::directory_options::skip_permission_denied)) {
```

Comment:
https://en.cppreference.com/w/cpp/filesystem/directory_iterator/directory_iterator
construct the iterator using (4) or (5), if it is end() - check the error code and return an error message.

I am also not sure if we want to skip permission denied here.